src/webapp/pages/NeighborhoodFinder.js

```diff
@@ -15,6 +15,37 @@ import QuadrantMap from "webapp/components/QuadrantMap";
 import SaveNeighborhoodDataInput from "webapp/components/SaveNeighborhoodDataInput";
 import sharedStylesFn from "webapp/style/sharedStyles";
 
+const getError = ({ responseError, responseData, str }) => {
+  if (responseError) {
+    return (
+      <>
+        {str("webapp:zoneFinder.geoError.message")}
+        &nbsp;
+        <a
+          href={str("webapp:slack.techChannelUrl")}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {str("webapp:slack.techChannel")}
+        </a>
+      </>
+    );
+  }
+
+  const addressIsOutsideDeliveryArea =
+    responseData && responseData.quadrant === null;
+  if (addressIsOutsideDeliveryArea) {
+    return str(
+      "webapp:zoneFinder.message.reminder",
+      `Reminder: the boundaries on this map show a defined delivery area, 
```

Comment:
to make the error message clearer, i think we can update it to `This address is outside of CHMAs delivery area. Per our Delivery Area proposal that passed 7/25. CHMA cannot make any deliveries outside of this area, no exceptions.`